public/static/locales/en/form-builder.json

```diff
@@ -329,5 +329,12 @@
   "translateTitle": "Enter your translation",
   "unlockPublishing": "Unlock publishing",
   "unlockPublishingDescription": "To unlock publishing, provide the contact information of the person you report to.",
-  "untitledForm": "Unnamed form file"
+  "untitledForm": "Untitled form",
```

Comment:
This should be "Unnamed form file" --- looks like that update was reverted here

Reply:
Thanks will fix that merge conflict